docs/usage.md

```diff
@@ -263,4 +268,34 @@ Following section explains how to get a list of closed change requests (CRs) sin
 
 2. Click **Apply Latest Content** tile under the **Production** tab.
 
-3. Click **Fetch Latest Changes** button from the lower-left of the screen to get the list of issues resolved since the last applied change.
\ No newline at end of file
+3. Click **Fetch Latest Changes** button from the lower-left of the screen to get the list of issues resolved since the last applied change.
+
+## Known Issues
+
+Renaming a record, after raising the PR, creates a duplicate record with the same UUID when the changes are pushed through the PR. This action results in the import process (*Apply Latest Content*) failing.
```

Comment:
Suggestion: Renaming a record in FortiSOAR, after raising the PR, creates a duplicate file/folder with the same UUID results in the import process......

Reply:
Changed. Did not add "FortiSOAR" as these actions are specific to FortiSOAR application.